README.md

```diff
@@ -30,7 +30,10 @@ cd nyrkio
 git submodule init
 git submodule update
 
-cat <<<END  > .env.backend
+cat > .env.backend << END
+## When using docker-compose.dev.yml with MongoDB running as a container.
+#DB_URL=mongodb://mongodb:27017/mongodb
+
 #DB_URL=mongodb://localhost:27017/nyrkiodb
```

Comment:
I think we should just make running mongo in a container the default and remove all other `DB_URL` examples.